osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -106,6 +111,15 @@ protected void UpdateFailed()
         }
 
         /// <summary>
+        /// Notifies subscribers of <see cref="NewJudgement"/> that a new judgement has occurred.
+        /// </summary>
+        /// <param name="judgement">The judgement to notify subscribers of.</param>
+        protected void NotifyNewJudgement(Judgement judgement)

```

Comment:
Called from the derived `ScoreProcessor<TJudgement>` because onJudgement can only exist at a `ScoreProcessor<TJudgement>`, but externals only have access to `ScoreProcessor`.
Following that I made NewJudgement an event in-case any other externals want to use it.